.rubocop.yml

```diff
@@ -1,12 +1,13 @@
 require:
-  - rubocop-rails
-  - rubocop-performance
-  - rubocop-rspec
   - rubocop-capybara
-  - rubocop-rspec
   - rubocop-rspec_rails
   - rubocop-factory_bot
 
+plugins:
+  - rubocop-performance
+  - rubocop-rails
+  - rubocop-rspec
```

Comment:
gets rid of a deprecation warning that was added in 1.72.x.  i confirmed that this works in general in other projects by making this change first, and continuing to get rubocop rails errors before making the fixes rubocop suggested (see e.g. [this week's pres cat dep update PR](https://github.com/sul-dlss/preservation_catalog/pull/2385), where i made those changes as separate commits in order to better show that).